hsctl

```diff
@@ -33,6 +33,8 @@ PROD_SSH_COMMAND='# REMOVED SSH COMPONENT'
 
 USE_LOCAL_IRODS=false
 
+source ~/hydroshare/hsctl_node
```

Comment:
This is a pretty big change, hsctl_node and hsctl_dev define a lot of methods.  I assume this is @dan-palmer1's work and it really should be in it's own review.  I'd like to to review it along with @cbcunc and @alvacouch

Reply:
Please work directly with Dan, Chris, Alva and Martin on this. I have not been involved here, other than designing the requirement a few months ago, that Discover must be able to deploy to local developer environments and hs-dev, beta, prod machines with automation. It looks like .hsctl was the selected solution.

Reply:
This should be moved to a separate PR, as I had originally suggested.